client/components/shared/ImportModule.js

```diff
@@ -8,7 +8,7 @@ export default class ImportModule extends React.Component {
 
         let $importFile = document.getElementById('importFile');
         if (!$importFile || !$importFile.files || !$importFile.files.length) {
-            alert('Need to select a file!');
+            alert('No file selected');
```

Comment:
Done. On a side note, once I'm through with revamping categories and allowing multiple selection, I think I will focus on integrating alerts in panels instead of displaying raw alert boxes.